freshmaker/image_verifier.py

```diff
@@ -147,21 +139,22 @@ def verify_image(self, image_nvr):
         :rtype: dict
         :return: Dict with image NVR as key and list of content_sets as values.
         """
-        repo = self._get_repository_from_image(image_nvr)
-        self._verify_repository_data(repo)
+        images = self.pyxis.find_images_by_nvr(image_nvr)
 
-        images = self.lb.get_images_by_nvrs([image_nvr], include_rpm_manifest=False)
         if not images:
+            raise ValueError("No images found for the specified NVR")
+
+        if len(images) != 1:
             raise ValueError(
-                "No published images tagged by %r found in repository" % (
-                    repo["auto_rebuild_tags"]))
+                "Multiple instances of the image with `%s` nvr found in Pyxis." % images[0]["brew"]["build"])
```

Comment:
@qixiang Does this make sense?

Reply:
The old LightBlue API (`find_container_images`) returns a list of `ContainerImage` and each of that represent an image with data for all available architectures, in other words, LightBlue returns a list of images (for a specific NVR) and then it was converted to a single `ContainerImage` with content sets from all architectures.

Our Pyxis API (`find_images_by_nvr`) returns a list of images, in most cases, the `len(images)` will be >1 because we didn't convert images of an NVR to a single object. We can verify each of images in this case:
```
content_sets = set()
for image in images:
    self._verify_image_data(image)
    content_sets.update(image["content_sets"])
...

return {images[0]["brew"]["build"]: sorted(content_sets)}
```